### ISSUE TITLE
hitting the adjust suit sensors verb gives you a menu containing all nearby mobs and yourself

### ISSUE THREAD

**Issue by newfren** [tgstation/tgstation#69883]
Reporting client version: 514.1585

## Round ID:
[190410](https://scrubby.melonmesa.com/round/190410)

## Testmerges:
- [Micros the lighting subsystem (Saves a second of init)](https://www.github.com/tgstation/tgstation/pull/69838)
- [Removes overlay queuing, saves 6/7 seconds of initialize. Lightly modifies stat tracking macros](https://www.github.com/tgstation/tgstation/pull/69696)
- [Macro optimizes SSmapping saving 50% ](https://www.github.com/tgstation/tgstation/pull/69632)
- [Completely refactors hallucinations, and also adds a few ](https://www.github.com/tgstation/tgstation/pull/69706)

## Reproduction:
![image](https://user-images.githubusercontent.com/12635164/189891007-00e81747-6f83-4b21-b322-e0d747bee302.png)
click adjust suit sensors in the object tab in the top right
get this menu
click yourself
get the regular menu

right clicking your suit functions as usual

if one of the mobs is another player you can't adjust their sensors

**Comment by ShizCalev** [comment]
Wat

**Comment by lessthnthree** [comment]
From my duplicate ticket. Not sure if it's related to #69811 and #69810 but got this in, yeah.:

I was fighting an EMP blob and went down about three times, and was restrained twice to get carried out of the fight. When I made sure to check my sensors after fighting the blob I noticed the normal sensors were not present, instead I could set my sensors to the names of people around me. Personally set my sensors to McGriff, and it appears that this carries to other unforms as well.
UPDATE: So I just realized this is a menu for adjusting the suit sensors of other people around me, when I click my own name it then allows me to change my suit sensors.

![image](https://user-images.githubusercontent.com/83487515/190100344-1e7411ce-27bc-4b52-836f-5f884e9914e1.png)
